week-2-practise/level-3/2-validation.js

```diff
@@ -6,7 +6,31 @@
 
 var excludedNums = [6, 14, 91, 111];
 
-function validate(num) {}
+  function validate(num) {
+    if(typeof(num) == 'number'){
+      return isFloat(num);
+    } else{
+      return false;
+    }
+  }
+
+  function isFloat(number){
```

Comment:
Same comment as above. Look at how the method names and the logic of that method don't match here. Can you think of a way to fix that?